core/src/main/java/io/aiven/klaw/config/ManageDatabase.java

```diff
@@ -259,6 +262,34 @@ private void loadStaticDataToDb() throws KlawException {
     }
   }
 
+  private void setDefaultEntitySequencesForTenant(int tenantId) {
```

Comment:
similar to this, migrations (inserts) done in MigrateData2x4x0 should be updated may be DefaultDataService ?

Reply:
Need to check them out, if an insert is the addition of new config data should be placed here if it is the migration of data from one location to the next we cna keep where it is. Will look at that now.